css/main.css

```diff
@@ -51,6 +52,68 @@ header {
   color: white;
 }
 
+.mobile-menu {
+  background-color: #3c3a39;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  transform: translateX(-100%);
+  transition: transform 0.3s ease-in-out;
+}
+
+.close-menu {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  background: transparent;
+  border: none;
+  outline: none;
+  font-size: 30px;
+  cursor: pointer;
+  color: gray;
+}
+
+.mobile-menu-list {
+  list-style: none;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+li {
+  list-style: none;
```

Comment:
- [ ] [OPTIONAL] According to CSS best practices, I think it would be better to remove this line as ```list-style``` to avoid duplicate code.